science/memesuite/Portfile

```diff
@@ -21,11 +20,13 @@ homepage            https://meme-suite.org
 master_sites        ${homepage}/meme/meme-software/${version}
 distname            meme-${version}
 
-checksums           rmd160  a068ad5b3cc70149477d3b0d7f00e231a628f865 \
-                    sha256  bc6f693bf5d81778b4e2901c2164c35c68f42ed080e2401b6e22960bf039a272 \
-                    size    52764373
+checksums           rmd160  e5f56d338b3b69257c03edbe2f34cea7b6e04577 \
+                    sha256  d7369cb01b55db477b7f73be1cf431d1ae86e92f14786e9891b9a9dff2b0975a \
+                    size    52798604
 
 set perl_vers       5.34
+set python_vers     3.11
+set python_vs       311
```

Comment:
it would be better to have the version number coded only once and set the other one, for example as in:
```
set py_ver          3.11
set py_ver_nodot    [string map {. {}} ${py_ver}]
```

Reply:
Hi,  I made the above changes in my local master and pushed it.  I'm afraid I do not know how fix the fact that there are now 3 commits.  Can you advise me what to do next?

Reply:
Normally you should rebase and force-push to consolidate your changes into a single commit. However we can fix it on our end using the "Rebase and merge" button.

Normally you should make your proposed changes in a branch, not your master. That way there will not be problems if you wish to submit proposals for multiple independent ports in multiple PRs.